File: src/test/java/com/converter/api/model/UserTest.java
package com.converter.api.model;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class UserTest {

    @Test
    void mustEncryptThePassword(){

        User user = new User("a@a.com", "12345678");
        Assertions.assertNotEquals("12345678", user.getPassword());
    }

}

File: src/test/kotlin/com/converter/api/model/UserTest.kt
package com.converter.api.model

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class UserTest {

    @Test //é pra falhar mesmo
    fun mustEncryptThePassword() {
        val user = User(
            email = "a@a.com",
            password = "12345678"
        )
        Assertions.assertNotEquals("12345678", user.password)
    }
}